src/prefect/cli/shell.py

```diff
@@ -91,6 +91,7 @@ def run_shell_process(
         text=True,
```

Comment:
I think we should move the existing hard-coded kwargs to defaults on the `popen_kwargs` dict to avoid user-provided kwargs conflicting with the internally used ones.

Reply:
ahh thats a good point - will update